shared-data/default-theme/html/jsapi/ui/topbar.js

```diff
@@ -31,44 +31,60 @@ $(document).on('click', '#form-search .clear-search', function(e) {
   $(e.target).hide();
 });
 
+/* Search - Special handling of certain queries */
+$(document).on('submit', '#form-search', function(event) {
+  var commands = {
+    in: function(cmdArgs) {
```

Comment:
This is unnecessary - in:inbox is a search term and all /in/tagname/ does is invoke the search function with in:tagname as a parameter. The only difference is the URL in the location bar, which isn't worth adding the maintenance burden of trying to figure out which tags exist at this point in the Javascript. :)